process/block/shardblock.go

```diff
@@ -40,6 +40,13 @@ type shardProcessor struct {
 	core          serviceContainer.Core
 	txCoordinator process.TransactionCoordinator
 	txCounter     *transactionCounter
+
+	appStatusHandler core.AppStatusHandler
```

Comment:
should be instantiated with NilAppStatusHandler in the constructor

Reply:
done